(dev/core#3844) Dummy payment processor should be flagged as such on …

yashodha opened the pull request:
…LIVE page

Overview
----------------------------------------
When dummy payment processor is configured for contribution pages/ event, there is no indication that is NOT a real transaction. This could be confusing and we should flag any LIVE page that is configured with dummy processor.

Before
----------------------------------------
No indication

After
----------------------------------------
![ec](https://user-images.githubusercontent.com/3455173/210722909-8ce1f654-3e16-493f-b9b5-45538894f16b.png)

yashodha committed:
(dev/core#3844) Dummy payment processor should be flagged as such on LIVE page

demeritcowboy commented:
Unrelated test fail.

I like this and it works well, just might suggest fixing the missing "a" in the grammar: "This page is currently configured with _a_ test payment processor". Or maybe instead of $dummy being a number it could be the title of the payment processor, and then output that, like `{ts 1="$dummyTitle|escape"}This page is currently configured to use the %1 payment processor. ...{/ts}

yashodha commented:
@demeritcowboy Thanks for your suggestions. I have incorporated the same and updated the PR. Can you please check?
![Screenshot from 2023-01-09 19-39-58](https://user-images.githubusercontent.com/3455173/211327458-f38e2dc8-bf03-4639-9cb1-7228cc8b6f0e.png)

demeritcowboy commented:
Looks good!
